clients/unshaded/src/main/java/tachyon/client/block/BlockStoreContext.java

```diff
@@ -156,6 +157,31 @@ public synchronized BlockWorkerClient acquireWorkerClient(String hostname) throw
   }
 
   /**
+   * Obtains a worker client to the worker with the given address.
+   *
+   * @param address the address of the worker to get a client to
+   * @return a {@link BlockWorkerClient} connected to the worker with the given hostname
+   * @throws IOException if no Tachyon worker is available for the given hostname
+   */
+  public synchronized BlockWorkerClient acquireWorkerClient(WorkerNetAddress address)
```

Comment:
What is the purpose for all of the acquire*WorkerClient() methods?

- acquireWorkerClient(WorkerNetAddress)
- acquireLocalWorkerClient()
- acquireRemoteWorkerClient(String)
- acquireRemoteWorkerClient(WorkerNetAddress)

Who calls which ones, and do we need all the variations?